cmd/stellar-rpc/internal/config/options.go

```diff
@@ -62,6 +61,7 @@ func (cfg *Config) options() Options {
 			Usage:     "Admin endpoint to listen and serve on. WARNING: this should not be accessible from the Internet and does not use TLS. \"\" (default) disables the admin server",
 			ConfigKey: &cfg.AdminEndpoint,
 		},
+		// TODO: should be gave a similar stellar-core-query-url parameter or should we assume that all queries will happen to the local captive core?
```

Comment:
You can't enable it on a validator and we def don't want another core running, so yeah it should be on the same captive core :+1:

Reply:
I didn't know it couldn't be enabled it on validators.